### PR TITLE
Document the database package development flow

### PR DESCRIPTION
**Pull request by whereswaldon**
/assign @jeefy

### CODE REVIEW

File: pkg/db/db_helpers_test.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func getDBConfig(t *testing.T) dbConfig {
 	if dbPool != nil {
 		const password = "secret"
 		// pulls an image, creates a container based on it and runs it
-		resource, err := dbPool.Run("postgres", "13", []string{"POSTGRES_PASSWORD=" + password})
```
Comment (jeefy) [inline pkg/db/db_helpers_test.go:57]: Was this change intentional?

Comment (whereswaldon) [reply pkg/db/db_helpers_test.go:57]: Yup, this is the version we have in RDS. I noticed the mismatch and corrected it, but only mentioned that in a commit message.

Comment (jeefy) [reply pkg/db/db_helpers_test.go:57]: :+1:
